x-pack/plugins/uptime/public/components/common/step_detail_link.tsx

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC } from 'react';
+import { EuiLink } from '@elastic/eui';
+import { Link } from 'react-router-dom';
+
+interface StepDetailLinkProps {
+  /**
+   * The ID of the check group (the journey run)
+   */
+  checkGroupId: string;
+  /**
+   * The index of the step
+   */
+  stepIndex: number;
+}
+
+export const StepDetailLink: FC<StepDetailLinkProps> = ({ children, checkGroupId, stepIndex }) => {
+  const to = `/journey/${checkGroupId}/step/${stepIndex}`;
+
+  return (
+    <EuiLink>
+      <Link data-test-subj={`step-detail-link`} to={to}>
+        {children}
+      </Link>
+    </EuiLink>
+  );
```

Comment:
This kills the purpose of EuiLink, we have a small utility for these kind of use cases,

can you please use that https://github.com/elastic/kibana/blob/master/x-pack/plugins/uptime/public/components/common/react_router_helpers/link_for_eui.tsx#L55

Reply:
Sure, I can change it. I'd actually copied this style from here: https://github.com/elastic/kibana/blob/master/x-pack/plugins/uptime/public/components/common/monitor_page_link.tsx#L22, so maybe that needs changing too?

Reply:
yeah we should change it there as well. i guess we can create a follow up issue to refactor all places in uptime.